backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -289,9 +289,10 @@ export class ExperimentAssignmentService {
     }
 
     // 6. Storing new/updated monitored decision point document
-    const assignmentUnit = experiments
-      ? experiments.find((exp) => exp.id === experimentId)?.assignmentUnit || experiments[0]?.assignmentUnit
-      : null;
+    const assignmentUnit =
```

Comment:
is this supposed to be in here?

Reply:
Small code improvement I added as a test to force the jobs to not be skipped. It makes the that variable assignment more consistent. I can remove it if you'd like.